### PR TITLE
Service definitions for Spark Master and Spark Worker

### PR DESCRIPTION
**Pull request by wolf31o2**
This, along with Spark HistoryServer, allows for setting up a standalone Spark cluster, without needing to setup YARN or Mesos.

### CODE REVIEW

File: services/spark-master.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      "install": {
+        "type":"chef-solo",
+        "fields": {
+          "run_list": "recipe[hadoop::spark_master]"
+        }
+      },
+      "configure": {
+        "type": "chef-solo",
+        "fields": {
+          "run_list": "recipe[hadoop_wrapper::default],recipe[hadoop::spark]"
```
Comment (dereklwood) [inline services/spark-master.json:27]: looks like this should be `hadoop::spark_master` instead of `hadoop::spark`, since there is config in that recipe too.

File: services/spark-worker.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      "install": {
+        "type":"chef-solo",
+        "fields": {
+          "run_list": "recipe[hadoop::spark_worker]"
+        }
+      },
+      "configure": {
+        "type": "chef-solo",
+        "fields": {
+          "run_list": "recipe[hadoop_wrapper::default],recipe[hadoop::spark]"
```
Comment (dereklwood) [inline services/spark-worker.json:27]: same as above, this should be `hadoop::spark_worker` instead of just `hadoop::spark`

**Comment by dereklwood** [comment]
1 comment, otherwise :+1: lgtm

File: services/spark-master.json (hunk 1 above)
Comment (wolf31o2) [inline services/spark-master.json:27]: Changing this... this actually needs to change for every hadoop service, since they all use the new init templates.

Comment (wolf31o2) [inline services/spark-master.json:27]: Let me do that as another PR and change all of them at once.

**Comment by wolf31o2** [comment]
I am going to change the configure action on all of the templates in another PR.
